### PR TITLE
Fix network policies for AliCloud and Openstack

### PR DESCRIPTION
**Pull request by mvladev**
**What this PR does / why we need it**:

Adds Openstack and AliCloud network policies labels to components deployed next to the K8S control plane.

**Which issue(s) this PR fixes**:

n/a

**Special notes for your reviewer**:

See https://github.com/gardener/gardener/pull/904

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: controllers/provider-openstack/charts/internal/cloud-controller-manager/templates/cloud-controller-manager.yaml [hunk 1]
```diff
@@ -91,6 +96,10 @@ spec:
             initialDelaySeconds: 15
             periodSeconds: 10
             timeoutSeconds: 15
+          ports:
+          - containerPort: {{ include "cloud-controller-manager.port" . }}
```
Comment (rfranzke) [inline controllers/provider-openstack/charts/internal/cloud-controller-manager/templates/cloud-controller-manager.yaml]: Can you add/define this? It's not yet part of this chart

Comment (mvladev) [reply controllers/provider-openstack/charts/internal/cloud-controller-manager/templates/cloud-controller-manager.yaml]: https://github.com/gardener/gardener-extensions/blob/6c6f41e20857c06ffb07488ea952485e7870f015/controllers/provider-openstack/charts/internal/cloud-controller-manager/templates/_helpers.tpl#L7-L13

**Comment by rfranzke** [comment]
needs rebase

**Comment by rfranzke** [comment]
also, run `make generate`

**Comment by mvladev** [comment]
Rebased to latest changes.

**Comment by rfranzke** [comment]
You have to run `make generate`. Also, please squash the commits

**Pull request merged**